.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,69 @@
+name: Integration Testing  
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: cs9 x86_64 e2e 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y install podman 
+
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          # Use githash of a tested commit instead of merge commit
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: build hirte tests containers
+        run: |
+          podman run --rm --systemd=always --name hirte-builder \
+          -v  ${PWD}:/hirte:Z -it quay.io/yarboa/hirte-builder:1.0.0 \
+          /bin/bash -c "rm -rf builddir; meson setup builddir; \
+          meson install -C builddir --destdir bin"
+          podman build -f tests/Containerfile -t hirte-image .
+
+      - name: run hirte containers
+        run: |
+          podman run -d --rm -p 2020:2020 --systemd=always --name hirte-mgr \
```

Comment:
Why are you running with --systemd=always?  If your container defaults to running systemd, then Podman will pick this up.  --systemd=always is usually just used for debugging what is going on in a systemd environment and should not be used in general.

Reply:
Ack